docs(tracing): remove patch_all from basic usage docs [3.0] (#11868)
DataDog/dd-trace-py: docs/basic_usage.rst

Basic Usage
===========

Automatic Instrumentation
~~~~~~~~~~~~~~~~~~~~~~~~~

``ddtrace.auto``
----------------

> To enable full ddtrace support (library instrumentations, profiling, application security monitoring, dynamic instrumentation, etc.) call :ref:`import ddtrace.auto<ddtraceauto>` as early as possible

Reviewer: ```suggestion
To enable full ddtrace support (library instrumentations, profiling, application security monitoring, dynamic instrumentation, etc.) call :ref:`import ddtrace.auto<ddtraceauto>` as the very first thing
```

> To enable full ddtrace support (library instrumentations, profiling, application security monitoring, dynamic instrumentation, etc.) call :ref:`import ddtrace.auto<ddtraceauto>` as the very first thing

Reviewer 2: ```suggestion
To enable full ddtrace functionality, including library instrumentations, profiling, application security monitoring, dynamic instrumentation, and more, call :ref:`import ddtrace.auto<ddtraceauto>` as the very first thing
```
